client/v3/concurrency/session.go

```diff
@@ -135,7 +139,7 @@ func WithLease(leaseID v3.LeaseID) SessionOption {
 // context is canceled before Close() completes, the session's lease will be
 // abandoned and left to expire instead of being revoked.
 func WithContext(ctx context.Context) SessionOption {
-	return func(so *sessionOptions) {
+	return func(so *sessionOptions, logger *zap.Logger) {
```

Comment:
Should I replace `logger` with `_` since this parameter is unused?

Reply:
yes